### DISCUSSION TITLE
Direct route links don't work for Vite React SPA

### DISCUSSION THREAD

**Discussion by srahimeen** [vercel/community#5976] [Help]
### Summary

I deployed a Vite React SPA to Vercel, and noticed that while the main URL of the app works, any direct route links don't work in the browser. 
For example `myapp.com` works, but `myapp.com/about` results in a 404.

![image](https://github.com/vercel/community/assets/9031145/480806af-c44b-493b-ad57-108e3004cdcf)

What do I need to do in order to get direct route links to work?

I know its possible to add nginx if I was using my own Dockerfile, but not sure how to achieve the same with Vercel's deployment.

I am using Google Chrome on Windows 10.

### Steps to Reproduce

1. Deploy a Vite React SPA with multiple routes to Vercel
2. Try to access a direct route such as `myapp.com/about`
3. Notice that you get a 404 error

**Comment by alexsvt2** [comment]
I'm having the same error message, did you solve your problem?
